package io.usoamic.validateutilkt.exception

class PrivateKeyRequiredError : ValidateUtilError()